app/controllers/miq_ae_class_controller.rb

```diff
@@ -1822,11 +1825,14 @@ def create_action_url(node)
 
   def get_rec_name(rec)
     column = rec.display_name.blank? ? :name : :display_name
-    if rec.kind_of?(MiqAeNamespace) && rec.domain? && (!rec.editable? || !rec.enabled)
-      add_read_only_suffix(rec, rec.send(column))
-    else
-      rec.send(column)
+    if rec.kind_of?(MiqAeNamespace) && rec.domain?
+      editable_domain = User.current_tenant.editable_domains.include?(rec)
```

Comment:
@h-kataria @kbrock @mkanoor shouldn't the rec know if it's editable based on your current tenant?  I don't follow why we go through the current tenant to get all of the editable domains to see if it includes the current domain... just ask the domain, right?  They include the tenancy mixin...  Just my 2 cents.